themes/default/views/pageFormat/pageHeader.php

```diff
@@ -110,6 +110,6 @@
 			<![endif]-->
 		<!-- super fish end menus -->
 	</head>	
-	<body>
+	<body <?php if(!in_array($this->request->getController(), array('Dashboard', 'Auth'))) :?>class="withSidebar"<?php endif?>>
 		<div id="leftNavBackground"></div>
-		<div align="center">
\ No newline at end of file
+		<div id="centered">
```

Comment:
changed to `<div class="centered">`, better than `<div align="center">` which was invalid